back/app/services/permissions/base_permissions_service.rb

```diff
@@ -90,7 +91,7 @@ def user_requirements_service
     end
 
     def user_can_moderate_something?(user)
-      @user_can_moderate_something ||= (user.admin? || UserRoleService.new.moderates_something?(user))
+      @user_can_moderate_something ||= user.admin? || UserRoleService.new.moderates_something?(user)
```

Comment:
Technically the brackets may not be needed here, but makes it more readable IMO

Reply:
I think this was done automatically by my linter